### PR TITLE
Create newsletter_21_Jan2020.md

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* Add the January 2021 newsletter draft

### List of changes proposed in this PR (pull-request)

* Created new file for the newsletter
* Add images

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: communications/newsletters/newsletter_21_Jan2020.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+### Weekly Coworking Calls
+We have also been hosting Coworking Calls every Monday and Thursday from 11:00 - 12:00 UTC. 
+To stay connected with people in Eastern Time Zones, we have added one more session on Wednesdays from 15:00 - 16:00 UTC, except when Collaboration Cafes are planned.
+Find the dates for the upcoming coworking calls on this shared note: [https://hackmd.io/@turingway/coworking-call](https://hackmd.io/@turingway/coworking-call). 
+
+### Join the Spanish Translation Team on Mondays
+Camila Rangel-Smith and Raina Camacho Toro are hosting coworking sessions every Monday at 17:00 UTC to work on the [Spanish translation of the book](https://www.transifex.com/theturingway/theturingway/).
+Connect with them on the Slack channel #traducción-español ([Slack invitation link](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw)) to receive rolling invites to join in their effort.
+
+## News from the community
```
Comment (Arielle-Bennett) [inline communications/newsletters/newsletter_21_Jan2020.md:44]: Not sure this heading quite fits based on the content below. It feels more like a section on other ways to contribute to the Turing Way - maybe "Other community contribution paths" / "Other community paths" / "More ways to support The Turing Way"?

Comment (malvikasharan) [reply communications/newsletters/newsletter_21_Jan2020.md:44]: That's a usual header where I post stuff from the community (GitHub and Slack posts). So, maybe for this month, I should change it to fit the details. Thank you for the suggestions.

**Comment by malvikasharan** [comment]
@Arielle-Bennett To fit the regular format, I have rephrased the news items under "News from the community" so that they actually look like news! I hope this makes more sense.
- Support for Representing The Turing Way at Conferences
- Opportunities to Lead Small Projects Within the Community
- ...

**Review by KirstieJane — APPROVED** [review]
Absolutely incredible. I’m blown away by how many amazing people we have in this community and everything they’ve built. GREAT WORK! 🌟🐙🌄💗

**Pull request merged**